summarization/stamp_page_picking/stamp_scores_dict.py

```diff
@@ -0,0 +1,17 @@
+''' Module to define and return the stamp costs dict '''
+
+from data_models.summarizer_output import StampPageType
+
+STAMP_SCORES = dict()
+
+STAMP_SCORES[StampPageType.TEXT_ONLY.name] = 1.0
+STAMP_SCORES[StampPageType.QUOTED.name] = 2.5
+STAMP_SCORES[StampPageType.EMBEDDED.name] = 5.0
+STAMP_SCORES[StampPageType.MEDIA_ONLY.name] = 7.5
+STAMP_SCORES[StampPageType.MEDIA_WITH_TEXT.name] = 10.0
+STAMP_SCORES[StampPageType.MEDIA_WITH_TEXT_AND_TITLE.name] = 20.0
+
+
+def get_stamp_page_score(stamp_page):
+    ''' returns the stamp score from the dict'''
+    return STAMP_SCORES[stamp_page.stamp_type.name]
```

Comment:
Instead of creating dict and managing all these, you should create a method in StampType enum `get_stamp_score()`
Remove this file entirely, its not needed.

Reply:
thanks for the suggestion. will update accordingly!